pkg/storage/postgresql/issues.go

```diff
@@ -134,6 +134,8 @@ func (db DB) IssuesSummary(filter storage.Filter) ([]model.IssueSummary, storage
 					{{ if eq .Status "FIXED" }}
```

Comment:
Shouldn't we ignore also the `FALSE POSITIVE` findings in the `{{ if eq .Status "OPEN" }}` branch?

Reply:
You are absolutely right. I fixed it now.